client/src/ui/Landingpage.tsx

```diff
@@ -0,0 +1,113 @@
+
+import "./styling/Landingpage.css";
+import Image from "next/image";
+import TextCarousel from "./lpTextCarousel";
+import { Typography } from "@mui/material";
+import Link from '@mui/material/Link';
+import { red } from "@mui/material/colors";
+
+
+
+function Home(){
+    return(
+        <div>
+            {/*Section 1 Block*/}
+            <div className="sec1-container">
+                <div className="sec1-box">
+
+                    <div className="sec1-text-wheel">

```

Comment:
Unecessary div. Place the styles within the TextCarousel component instead of doing it outside of the component